### ISSUE TITLE
Backups fail

### ISSUE THREAD

**Issue by lgarron** [joeyh/github-backup#32]
I try to give `github-backup` a chance every few months. At this point, I [need to export some private repos](https://twitter.com/lgarron/status/573589776659447808), and I don't trust myself to use the API and copy back up all the issue metadata myself. (The [`issues` endpoint](https://api.github.com/repos/octocat/Hello-World/issues) deceptively includes the body of the first comment of the issue but not the rest. I don't want to play endpoint whack-a-mole on a repo I need to delete.)

After a few hours of wrestling with `cabal` dependencies, I finally got `github-backup` to run again, and I get the same error as ever:

    github-backup: Backup may be incomplete; [x] requests failed

Is this common? How do I debug this without sinking another 5 hours into it?
(Or is there some other solution that makes it easy to back up metadata for an entire repo?)

**Comment by silky** [comment]
yeah, i noticed that it fails totally on private repos, and for public repos i get some failures but some things working.

maybe it's due to github api changes.

**Comment by rom1504** [comment]
I found https://github.com/josegonzalez/python-github-backup and it seems to work much better

**Comment by lgarron** [comment]
> I found https://github.com/josegonzalez/python-github-backup and it seems to work much better

Works excellently, thank you!
